exporter/src/main/as/flump/SwfTexture.as

```diff
@@ -135,8 +135,8 @@ public class SwfTexture
 
         // calculate derivative info
         _origin = new Point(-_visualBounds.x, -_visualBounds.y);
-        _w = _visualBounds.width;
-        _h = _visualBounds.height;
+        _w = Math.ceil(_visualBounds.width);
+        _h = Math.ceil(_visualBounds.height);
```

Comment:
This is a bugfix. Here the Number -> int rounding caused edges of images to be lost resulting sometimes in small visual artifacts. (for example look at the right edge of the Belle.fla character's hair)